pkg/broker/broker.go

```diff
@@ -485,7 +485,7 @@ func (a AnsibleBroker) Catalog() (*CatalogResponse, error) {
 }
 
 // Provision  - will provision a service
-func (a AnsibleBroker) Provision(instanceUUID uuid.UUID, req *ProvisionRequest, async bool,
+func (a AnsibleBroker) Provision(instanceUUID uuid.UUID, req *ProvisionRequest, async bool, username string,
```

Comment:
You will need to do the same for all the other actions to truely pass this for every method. 

@eriknelson @jmrodri What do you guys think of having him add the requesting user `UserInfo` to the provision request?

Reply:
@shawn-hurley ProvisionRequest sounds better, but it usually maps to the OSB ProvisionRequest. Unless we add it as part of the parameters, then I'd be okay with that.